### DISCUSSION TITLE
Build -> Core -> Print(Problem within the docker container used in build) - Build Failed

### DISCUSSION THREAD

**Discussion by shounakacharya** [Sunbird-Ed/Community#81] [Installation] [answered]
System Configuration Details:
1.	KP, DP, DB, Yarn, Jenkins, Other-Services: Ubuntu 18.04 LTS, 4vCPU, 16GB RAM, 250 GB HDD
2.	Google Kubernetes engine: 1.21.12-gke.2200 with 4 worker nodes. Note: Currently Sunbird is not compatible with K8s APIs  >= 1.22 and we have checked this
3.	All systems are running on GCP
4. Github Release Tag: release-3.8.0_RC1

@gandham-santhosh and  @keshavprasadms

Jenkins Logs:

Started by user [Admin Deployer](http://35.222.148.131:8080/user/admin)
Rebuilds build #[1](http://35.222.148.131:8080/job/Build/job/Core/job/Print/1)
Checking out git https://github.com/project-sunbird/print-service.git into /var/lib/jenkins/workspace/Build/Core/Print@script to read Jenkinsfile
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Core/Print@script/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/print-service.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/print-service.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/print-service.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-3.8.0_RC1^{commit} # timeout=10
 > git rev-parse release-3.8.0_RC1^{commit} # timeout=10
Checking out Revision 92807caa9b0ceb02087649ce01f369134f001917 (release-3.8.0_RC1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 92807caa9b0ceb02087649ce01f369134f001917 # timeout=10
Commit message: "Merge pull request #22 from keshavprasadms/release-3.8.0"
 > git rev-list --no-walk 92807caa9b0ceb02087649ce01f369134f001917 # timeout=10
Running in Durability level: MAX_SURVIVABILITY
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [Jenkins](http://35.222.148.131:8080/computer/(built-in)/) in /var/lib/jenkins/workspace/Build/Core/Print
[Pipeline] {
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (Checkout)
[Pipeline] echo
Found environment variable named hub_org with value as: shounakacharya
[Pipeline] cleanWs
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
[WS-CLEANUP] done
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/project-sunbird/print-service.git
 > git init /var/lib/jenkins/workspace/Build/Core/Print # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/print-service.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/print-service.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/project-sunbird/print-service.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > git rev-parse origin/release-3.8.0_RC1^{commit} # timeout=10
 > git rev-parse release-3.8.0_RC1^{commit} # timeout=10
Checking out Revision 92807caa9b0ceb02087649ce01f369134f001917 (release-3.8.0_RC1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 92807caa9b0ceb02087649ce01f369134f001917 # timeout=10
Commit message: "Merge pull request #22 from keshavprasadms/release-3.8.0"
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/Build/Core/Print/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/print-service.git # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/print-service.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress -- https://github.com/project-sunbird/print-service.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/refs/tags/release-3.8.0_RC1^{commit} # timeout=10
 > git rev-parse refs/tags/release-3.8.0_RC1^{commit} # timeout=10
Checking out Revision 92807caa9b0ceb02087649ce01f369134f001917 (refs/tags/release-3.8.0_RC1)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 92807caa9b0ceb02087649ce01f369134f001917 # timeout=10
Commit message: "Merge pull request #22 from keshavprasadms/release-3.8.0"
[Pipeline] echo
github_release_tag specified, building from github_release_tag: release-3.8.0_RC1
[Pipeline] echo
build_tag: release-3.8.0_RC1
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Build)
[Pipeline] echo
Environment will be : build
[Pipeline] sh
+ chmod 777 build.sh
[Pipeline] sh
+ ./build.sh release-3.8.0_RC1 master shounakacharya
Sending build context to Docker daemon  174.6kB

Step 1/18 : FROM circleci/node:8.11.2-stretch as build
 ---> 3c168695e85e
Step 2/18 : MAINTAINER "Mahesh Kumar Gangula" "mahesh@ilimi.in"
 ---> Using cache
 ---> 8b66076a1075
Step 3/18 : USER root
 ---> Using cache
 ---> ceaa2818ed3c
Step 4/18 : COPY src /opt/print-service/
 ---> Using cache
 ---> ed94bd405c42
Step 5/18 : WORKDIR /opt/print-service/
 ---> Using cache
 ---> 37707e8af66e
Step 6/18 : RUN npm install --unsafe-perm
 ---> Using cache
 ---> a06f8e791e1f
Step 7/18 : FROM node:8.11-slim
 ---> 1aa312dac871
Step 8/18 : MAINTAINER "Mahesh Kumar Gangula" "mahesh@ilimi.in"
 ---> Using cache
 ---> 9ed110333dfe
Step 9/18 : RUN apt-get clean     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'     && apt update && apt install fonts-indic -y     && apt-get install -y google-chrome-unstable gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget       --no-install-recommends     && rm -rf /var/lib/apt/lists/*
 ---> Running in bf2383741653
OK

WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.

Get:1 http://security.debian.org/ jessie/updates InRelease [44.9 kB]
Ign http://deb.debian.org/ jessie InRelease
Get:2 http://deb.debian.org/ jessie-updates InRelease [16.3 kB]
Get:3 http://dl.google.com/ stable InRelease [1811 B]
Get:4 http://deb.debian.org/ jessie Release.gpg [1652 B]
Get:5 http://deb.debian.org/ jessie Release [77.3 kB]
Get:6 http://security.debian.org/ jessie/updates/main amd64 Packages [992 kB]
Get:7 http://deb.debian.org/ jessie-updates/main amd64 Packages [20 B]
Get:8 http://dl.google.com/ stable/main amd64 Packages [1096 B]
Get:9 http://deb.debian.org/ jessie/main amd64 Packages [9098 kB]
Fetched 10.2 MB in 3s (3031 kB/s)
Reading package lists...
Building dependency tree...
Reading state information...
37 packages can be upgraded. Run 'apt list --upgradable' to see them.

WARNING: apt does not have a stable CLI interface yet. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
The following extra packages will be installed:
  fonts-beng fonts-beng-extra fonts-deva fonts-deva-extra fonts-gargi
  fonts-gubbi fonts-gujr fonts-gujr-extra fonts-guru fonts-guru-extra
  fonts-kalapi fonts-knda fonts-lohit-beng-assamese fonts-lohit-beng-bengali
  fonts-lohit-deva fonts-lohit-gujr fonts-lohit-guru fonts-lohit-knda
  fonts-lohit-mlym fonts-lohit-orya fonts-lohit-taml
  fonts-lohit-taml-classical fonts-lohit-telu fonts-mlym fonts-nakula
  fonts-navilu fonts-orya fonts-orya-extra fonts-pagul fonts-sahadeva
  fonts-samyak-deva fonts-samyak-gujr fonts-samyak-mlym fonts-samyak-taml
  fonts-sarai fonts-smc fonts-taml fonts-telu fonts-telu-extra
The following NEW packages will be installed:
  fonts-beng fonts-beng-extra fonts-deva fonts-deva-extra fonts-gargi
  fonts-gubbi fonts-gujr fonts-gujr-extra fonts-guru fonts-guru-extra
  fonts-indic fonts-kalapi fonts-knda fonts-lohit-beng-assamese
  fonts-lohit-beng-bengali fonts-lohit-deva fonts-lohit-gujr fonts-lohit-guru
  fonts-lohit-knda fonts-lohit-mlym fonts-lohit-orya fonts-lohit-taml
  fonts-lohit-taml-classical fonts-lohit-telu fonts-mlym fonts-nakula
  fonts-navilu fonts-orya fonts-orya-extra fonts-pagul fonts-sahadeva
  fonts-samyak-deva fonts-samyak-gujr fonts-samyak-mlym fonts-samyak-taml
  fonts-sarai fonts-smc fonts-taml fonts-telu fonts-telu-extra
0 upgraded, 40 newly installed, 0 to remove and 37 not upgraded.
Need to get 3490 kB of archives.
After this operation, 12.3 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian/ jessie/main fonts-deva-extra all 3.0-2 [573 kB]
Get:2 http://deb.debian.org/debian/ jessie/main fonts-kalapi all 1.0-2 [116 kB]
Get:3 http://deb.debian.org/debian/ jessie/main fonts-smc all 6.0-1 [546 kB]
Get:4 http://deb.debian.org/debian/ jessie/main fonts-lohit-beng-bengali all 2.5.3-1 [78.9 kB]
Get:5 http://deb.debian.org/debian/ jessie/main fonts-lohit-beng-assamese all 2.5.3-1 [78.7 kB]
Get:6 http://deb.debian.org/debian/ jessie/main fonts-beng-extra all 1.0-4 [364 kB]
Get:7 http://deb.debian.org/debian/ jessie/main fonts-beng all 2:1.2 [2838 B]
Get:8 http://deb.debian.org/debian/ jessie/main fonts-lohit-deva all 2.5.3-1 [48.0 kB]
Get:9 http://deb.debian.org/debian/ jessie/main fonts-sahadeva all 1.0-3 [169 kB]
Get:10 http://deb.debian.org/debian/ jessie/main fonts-nakula all 1.0-3 [167 kB]
Get:11 http://deb.debian.org/debian/ jessie/main fonts-samyak-deva all 1.2.2-4 [54.6 kB]
Get:12 http://deb.debian.org/debian/ jessie/main fonts-deva all 2:1.2 [2906 B]
Get:13 http://deb.debian.org/debian/ jessie/main fonts-gargi all 2.0-3 [45.3 kB]
Get:14 http://deb.debian.org/debian/ jessie/main fonts-gubbi all 1.3-3 [60.8 kB]
Get:15 http://deb.debian.org/debian/ jessie/main fonts-lohit-gujr all 2.5.3-1 [38.4 kB]
Get:16 http://deb.debian.org/debian/ jessie/main fonts-samyak-gujr all 1.2.2-4 [43.7 kB]
Get:17 http://deb.debian.org/debian/ jessie/main fonts-gujr-extra all 1.0-3 [198 kB]
Get:18 http://deb.debian.org/debian/ jessie/main fonts-gujr all 2:1.2 [2858 B]
Get:19 http://deb.debian.org/debian/ jessie/main fonts-lohit-guru all 2.5.3-2 [20.4 kB]
Get:20 http://deb.debian.org/debian/ jessie/main fonts-guru-extra all 2.0-3 [52.8 kB]
Get:21 http://deb.debian.org/debian/ jessie/main fonts-guru all 2:1.2 [2824 B]
Get:22 http://deb.debian.org/debian/ jessie/main fonts-lohit-knda all 2.5.3-1 [59.0 kB]
Get:23 http://deb.debian.org/debian/ jessie/main fonts-navilu all 1.2-2 [77.7 kB]
Get:24 http://deb.debian.org/debian/ jessie/main fonts-knda all 2:1.2 [3292 B]
Get:25 http://deb.debian.org/debian/ jessie/main fonts-lohit-mlym all 2.5.4-1 [34.9 kB]
Get:26 http://deb.debian.org/debian/ jessie/main fonts-samyak-mlym all 1.2.2-4 [27.0 kB]
Get:27 http://deb.debian.org/debian/ jessie/main fonts-mlym all 2:1.2 [2894 B]
Get:28 http://deb.debian.org/debian/ jessie/main fonts-lohit-orya all 2.5.4.1-1 [48.7 kB]
Get:29 http://deb.debian.org/debian/ jessie/main fonts-orya-extra all 2.0-3 [85.2 kB]
Get:30 http://deb.debian.org/debian/ jessie/main fonts-orya all 2:1.2 [2844 B]
Get:31 http://deb.debian.org/debian/ jessie/main fonts-pagul all 1.0-7 [78.6 kB]
Get:32 http://deb.debian.org/debian/ jessie/main fonts-lohit-taml all 2.5.3-1 [36.7 kB]
Get:33 http://deb.debian.org/debian/ jessie/main fonts-samyak-taml all 1.2.2-4 [23.4 kB]
Get:34 http://deb.debian.org/debian/ jessie/main fonts-lohit-taml-classical all 2.5.3-2 [32.7 kB]
Get:35 http://deb.debian.org/debian/ jessie/main fonts-taml all 2:1.3 [3018 B]
Get:36 http://deb.debian.org/debian/ jessie/main fonts-lohit-telu all 2.5.3-1 [68.2 kB]
Get:37 http://deb.debian.org/debian/ jessie/main fonts-telu-extra all 2.0-3 [182 kB]
Get:38 http://deb.debian.org/debian/ jessie/main fonts-telu all 2:1.2 [2824 B]
Get:39 http://deb.debian.org/debian/ jessie/main fonts-indic all 2:1.2 [3200 B]
Get:40 http://deb.debian.org/debian/ jessie/main fonts-sarai all 1.0-1 [50.5 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 3490 kB in 5s (613 kB/s)
Selecting previously unselected package fonts-deva-extra.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 8008 files and directories currently installed.)
Preparing to unpack .../fonts-deva-extra_3.0-2_all.deb ...
Unpacking fonts-deva-extra (3.0-2) ...
Selecting previously unselected package fonts-kalapi.
Preparing to unpack .../fonts-kalapi_1.0-2_all.deb ...
Unpacking fonts-kalapi (1.0-2) ...
Selecting previously unselected package fonts-smc.
Preparing to unpack .../fonts-smc_6.0-1_all.deb ...
Unpacking fonts-smc (6.0-1) ...
Selecting previously unselected package fonts-lohit-beng-bengali.
Preparing to unpack .../fonts-lohit-beng-bengali_2.5.3-1_all.deb ...
Unpacking fonts-lohit-beng-bengali (2.5.3-1) ...
Selecting previously unselected package fonts-lohit-beng-assamese.
Preparing to unpack .../fonts-lohit-beng-assamese_2.5.3-1_all.deb ...
Unpacking fonts-lohit-beng-assamese (2.5.3-1) ...
Selecting previously unselected package fonts-beng-extra.
Preparing to unpack .../fonts-beng-extra_1.0-4_all.deb ...
Unpacking fonts-beng-extra (1.0-4) ...
Selecting previously unselected package fonts-beng.
Preparing to unpack .../fonts-beng_2%3a1.2_all.deb ...
Unpacking fonts-beng (2:1.2) ...
Selecting previously unselected package fonts-lohit-deva.
Preparing to unpack .../fonts-lohit-deva_2.5.3-1_all.deb ...
Unpacking fonts-lohit-deva (2.5.3-1) ...
Selecting previously unselected package fonts-sahadeva.
Preparing to unpack .../fonts-sahadeva_1.0-3_all.deb ...
Unpacking fonts-sahadeva (1.0-3) ...
Selecting previously unselected package fonts-nakula.
Preparing to unpack .../fonts-nakula_1.0-3_all.deb ...
Unpacking fonts-nakula (1.0-3) ...
Selecting previously unselected package fonts-samyak-deva.
Preparing to unpack .../fonts-samyak-deva_1.2.2-4_all.deb ...
Unpacking fonts-samyak-deva (1.2.2-4) ...
Selecting previously unselected package fonts-deva.
Preparing to unpack .../fonts-deva_2%3a1.2_all.deb ...
Unpacking fonts-deva (2:1.2) ...
Selecting previously unselected package fonts-gargi.
Preparing to unpack .../fonts-gargi_2.0-3_all.deb ...
Unpacking fonts-gargi (2.0-3) ...
Selecting previously unselected package fonts-gubbi.
Preparing to unpack .../fonts-gubbi_1.3-3_all.deb ...
Unpacking fonts-gubbi (1.3-3) ...
Selecting previously unselected package fonts-lohit-gujr.
Preparing to unpack .../fonts-lohit-gujr_2.5.3-1_all.deb ...
Unpacking fonts-lohit-gujr (2.5.3-1) ...
Selecting previously unselected package fonts-samyak-gujr.
Preparing to unpack .../fonts-samyak-gujr_1.2.2-4_all.deb ...
Unpacking fonts-samyak-gujr (1.2.2-4) ...
Selecting previously unselected package fonts-gujr-extra.
Preparing to unpack .../fonts-gujr-extra_1.0-3_all.deb ...
Unpacking fonts-gujr-extra (1.0-3) ...
Selecting previously unselected package fonts-gujr.
Preparing to unpack .../fonts-gujr_2%3a1.2_all.deb ...
Unpacking fonts-gujr (2:1.2) ...
Selecting previously unselected package fonts-lohit-guru.
Preparing to unpack .../fonts-lohit-guru_2.5.3-2_all.deb ...
Unpacking fonts-lohit-guru (2.5.3-2) ...
Selecting previously unselected package fonts-guru-extra.
Preparing to unpack .../fonts-guru-extra_2.0-3_all.deb ...
Unpacking fonts-guru-extra (2.0-3) ...
Selecting previously unselected package fonts-guru.
Preparing to unpack .../fonts-guru_2%3a1.2_all.deb ...
Unpacking fonts-guru (2:1.2) ...
Selecting previously unselected package fonts-lohit-knda.
Preparing to unpack .../fonts-lohit-knda_2.5.3-1_all.deb ...
Unpacking fonts-lohit-knda (2.5.3-1) ...
Selecting previously unselected package fonts-navilu.
Preparing to unpack .../fonts-navilu_1.2-2_all.deb ...
Unpacking fonts-navilu (1.2-2) ...
Selecting previously unselected package fonts-knda.
Preparing to unpack .../fonts-knda_2%3a1.2_all.deb ...
Unpacking fonts-knda (2:1.2) ...
Selecting previously unselected package fonts-lohit-mlym.
Preparing to unpack .../fonts-lohit-mlym_2.5.4-1_all.deb ...
Unpacking fonts-lohit-mlym (2.5.4-1) ...
Selecting previously unselected package fonts-samyak-mlym.
Preparing to unpack .../fonts-samyak-mlym_1.2.2-4_all.deb ...
Unpacking fonts-samyak-mlym (1.2.2-4) ...
Selecting previously unselected package fonts-mlym.
Preparing to unpack .../fonts-mlym_2%3a1.2_all.deb ...
Unpacking fonts-mlym (2:1.2) ...
Selecting previously unselected package fonts-lohit-orya.
Preparing to unpack .../fonts-lohit-orya_2.5.4.1-1_all.deb ...
Unpacking fonts-lohit-orya (2.5.4.1-1) ...
Selecting previously unselected package fonts-orya-extra.
Preparing to unpack .../fonts-orya-extra_2.0-3_all.deb ...
Unpacking fonts-orya-extra (2.0-3) ...
Selecting previously unselected package fonts-orya.
Preparing to unpack .../fonts-orya_2%3a1.2_all.deb ...
Unpacking fonts-orya (2:1.2) ...
Selecting previously unselected package fonts-pagul.
Preparing to unpack .../fonts-pagul_1.0-7_all.deb ...
Unpacking fonts-pagul (1.0-7) ...
Selecting previously unselected package fonts-lohit-taml.
Preparing to unpack .../fonts-lohit-taml_2.5.3-1_all.deb ...
Unpacking fonts-lohit-taml (2.5.3-1) ...
Selecting previously unselected package fonts-samyak-taml.
Preparing to unpack .../fonts-samyak-taml_1.2.2-4_all.deb ...
Unpacking fonts-samyak-taml (1.2.2-4) ...
Selecting previously unselected package fonts-lohit-taml-classical.
Preparing to unpack .../fonts-lohit-taml-classical_2.5.3-2_all.deb ...
Unpacking fonts-lohit-taml-classical (2.5.3-2) ...
Selecting previously unselected package fonts-taml.
Preparing to unpack .../fonts-taml_2%3a1.3_all.deb ...
Unpacking fonts-taml (2:1.3) ...
Selecting previously unselected package fonts-lohit-telu.
Preparing to unpack .../fonts-lohit-telu_2.5.3-1_all.deb ...
Unpacking fonts-lohit-telu (2.5.3-1) ...
Selecting previously unselected package fonts-telu-extra.
Preparing to unpack .../fonts-telu-extra_2.0-3_all.deb ...
Unpacking fonts-telu-extra (2.0-3) ...
Selecting previously unselected package fonts-telu.
Preparing to unpack .../fonts-telu_2%3a1.2_all.deb ...
Unpacking fonts-telu (2:1.2) ...
Selecting previously unselected package fonts-indic.
Preparing to unpack .../fonts-indic_2%3a1.2_all.deb ...
Unpacking fonts-indic (2:1.2) ...
Selecting previously unselected package fonts-sarai.
Preparing to unpack .../fonts-sarai_1.0-1_all.deb ...
Unpacking fonts-sarai (1.0-1) ...
Setting up fonts-deva-extra (3.0-2) ...
Setting up fonts-kalapi (1.0-2) ...
Setting up fonts-smc (6.0-1) ...
Setting up fonts-lohit-beng-bengali (2.5.3-1) ...
Setting up fonts-lohit-beng-assamese (2.5.3-1) ...
Setting up fonts-beng-extra (1.0-4) ...
Setting up fonts-beng (2:1.2) ...
Setting up fonts-lohit-deva (2.5.3-1) ...
Setting up fonts-sahadeva (1.0-3) ...
Setting up fonts-nakula (1.0-3) ...
Setting up fonts-samyak-deva (1.2.2-4) ...
Setting up fonts-deva (2:1.2) ...
Setting up fonts-gargi (2.0-3) ...
Setting up fonts-gubbi (1.3-3) ...
Setting up fonts-lohit-gujr (2.5.3-1) ...
Setting up fonts-samyak-gujr (1.2.2-4) ...
Setting up fonts-gujr-extra (1.0-3) ...
Setting up fonts-gujr (2:1.2) ...
Setting up fonts-lohit-guru (2.5.3-2) ...
Setting up fonts-guru-extra (2.0-3) ...
Setting up fonts-guru (2:1.2) ...
Setting up fonts-lohit-knda (2.5.3-1) ...
Setting up fonts-navilu (1.2-2) ...
Setting up fonts-knda (2:1.2) ...
Setting up fonts-lohit-mlym (2.5.4-1) ...
Setting up fonts-samyak-mlym (1.2.2-4) ...
Setting up fonts-mlym (2:1.2) ...
Setting up fonts-lohit-orya (2.5.4.1-1) ...
Setting up fonts-orya-extra (2.0-3) ...
Setting up fonts-orya (2:1.2) ...
Setting up fonts-pagul (1.0-7) ...
Setting up fonts-lohit-taml (2.5.3-1) ...
Setting up fonts-samyak-taml (1.2.2-4) ...
Setting up fonts-lohit-taml-classical (2.5.3-2) ...
Setting up fonts-taml (2:1.3) ...
Setting up fonts-lohit-telu (2.5.3-1) ...
Setting up fonts-telu-extra (2.0-3) ...
Setting up fonts-telu (2:1.2) ...
Setting up fonts-indic (2:1.2) ...
Setting up fonts-sarai (1.0-1) ...
Reading package lists...
Building dependency tree...
Reading state information...
libc6 is already the newest version.
libc6 set to manually installed.
ca-certificates is already the newest version.
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 google-chrome-unstable : Depends: libdrm2 (>= 2.4.60) but 2.4.58-2 is to be installed
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c apt-get clean     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'     && apt update && apt install fonts-indic -y     && apt-get install -y google-chrome-unstable gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget       --no-install-recommends     && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 100
Finished: FAILURE

**Comment by beepdot** [comment, marked as answer]
Please change [this line](https://github.com/project-sunbird/print-service/blob/release-3.8.0/Dockerfile#L8) to `FROM node:8.11.4-stretch` and run the build. It should work.

**Reply by shounakacharya** [reply]
@keshavprasadms: Thanks. This worked. Marking as answered!

**Reply by dikshauat** [reply]
Hi @keshavprasadms @shounakacharya 
please let me know how can I make the above suggested changes

**Reply by beepdot** [reply]
Please go through the entire post and make the changes. All the steps are already mentioned.
